client/landing/jetpack-cloud/sections/backups/download/error.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Props {
+	error?: string;
+	siteTitle: string | null;
```

Comment:
What is the difference between `propName?: string;` and `propName: string | null;`?

Reply:
`undefined` vs `null`, in other words `propName?: string;` is equivalent to `propName: string | undefined;`

Reply:
but there is no syntactic sugar for a nullable prop ( that I am aware of )